src/Analyzer.Core.BuiltInRuleTests/Tests/AppService_EnableDiagLogs.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "Template": "AppServicesLogs", // didn't use the existing AppServices template because AppServiceWebApp_HTTPS would trigger for all the new resources, and AppService_EnableDiagLogs for all the already existing ones
```

Comment:
I personally think it is better to mesh them together so that the existing test results are not touched, but the resources work with this additional rule. I also see that when there are 500 rules, separate may be better and easier to work with, but more an issue for the future.

Reply:
To use the same file I would need to add extra properties to all the resources that were created for other rules so this one doesn't fail, and then add my resources, or try to mix some test cases editing the existing resources, but it wouldn't work for most and it would tangle both rules in a confusing way

Reply:
The tests were designed so you could use multiple template files for testing.  Did you try that?  Could that make it easier?

Reply:
As more tests (and therefore test template files) are added, my hope was that it would make writing tests easier, since you can just find templates that already fail the rule and refer to those lines in the test instead of making all new template files.

Reply:
I'm not sure how that would help in this case, I created a new template and didn't feel the need to use the resources in other templates

Reply:
I didn't consider that, I'll see if it helps, but I doubt it because the problem is precisely that when I used already existing templates it flagged too many resources, that wouldn't make that much sense to report in the specific rule test file

Reply:
Right now we are using the name of the properties to describe the resource. If we start using the same resources for multiple rules we'll have to use some other method to describe the rule. Maybe we can use the tags property in the resource and add a new tag for each scenario we're testing? This method would shift line numbers for other rules pretty often though.

Reply:
Sounds good to me.

Reply:
Also good with me.